Onboarding.md

```diff
@@ -18,7 +18,17 @@ Bonus videos:
 
 Our team has been working to extend the [CivicActions Accessibility](https://accessibility.civicactions.com/) site and integrate key elements into the [CivicActions Handbook](https://handbook.civicactions.com/). CivicActions believes in working in the open, so we also are leveraging tools on GitHub like the [Projects board](https://github.com/CivicActions/accessibility/projects/1) and [Discussions pages](https://github.com/CivicActions/accessibility/discussions).
 
-### Trello card - 
+
+### Trello card - install a web accessibility tool like [WebAim's WAVE Toolbar extension/plugin](https://wave.webaim.org/extension/)
+
+Installing an accessibility browser extension and [learning the basics of how to use it](https://www.youtube.com/watch?v=ITUDiTgAZY0&t=270s) is important for everyone creating web content.  
+
+
+### Trello card - investigate and possibly configure accessibility settings in your operating system
+
+Apple, Windows, Chromebook, Linux desktop OS all have built-in accessibility features. So do mobile systems like Android & iOS.
+
+**Mac users** - enable [Full Keyboard Access](https://www.macobserver.com/tips/quick-tip/navigate-mac-full-keyboard-access/#:~:text=%20To%20turn%20on%20Full%20Keyboard%20Access%3A%20,Keyboard.%204%20Click%20the%20Shortcuts%20tab.%20More%20) to be able to test keyboard only navigation.
```

Comment:
@mgifford looks like the link has that fancy google text syntax. Not a big deal but maybe we should drop it?

Reply:
What, you don't like supporting trackers? Are you really supporting the web my friend??

:)